presto-testing-kafka/src/main/java/io/prestosql/testing/kafka/TestingKafka.java

```diff
@@ -72,6 +72,32 @@ private void createTopic(int partitions, int replication, String topic)
         }
     }
 
+    public void createTopicWithConfig(int partitions, int replication, String topic, boolean enableLogAppendTime)
```

Comment:
Can you make `createTopic` call out to `createTopicWithConfig`?

Reply:
I'm not sure. It seems other module will having some test conflicts if using same createTopicWithConfig with identifying the local zookeeper config. So I make new one independently.